.env.example

```
@@ -22,3 +22,7 @@ BACKEND_S3_REGION=us-east-1
 BACKEND_S3_BUCKET=optimark-dev
 BACKEND_S3_ACCESS_KEY_ID=admin
 BACKEND_S3_SECRET_ACCESS_KEY=key
+BACKEND_AUTH_SESSION_COOKIE_NAME=optimark_session
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKEND_AUTH_SESSION_COOKIE_NAME key should go before the BACKEND_DATABASE_URL key

(UnorderedKey)